configFlannel.md

```diff
@@ -115,11 +119,15 @@ FLANNEL_OPTIONS="eth0"
 ```
 
 
-* Enable the flanneld service and reboot.
+* Enable the flanneld service, let systemd know that the local flanneld service needs a default route and the local etcd service to be started before it, and reboot.
 
 
 ```
-# systemctl enable flanneld
+# systemctl enable NetworkManager-wait-online.service
+# systemctl enable flanneld.service
+# mkdir /etc/systemd/system/flanneld.service.wants
+# ln -s /usr/lib/systemd/system/etcd.service /etc/systemd/system/flanneld.service.wants/
+# ln -s /usr/lib/systemd/system/NetworkManager-wait-online.service /etc/systemd/system/flanneld.service.wants/
```

Comment:
On Fri, 2015-06-26 at 07:07 -0700, Nalin Dahyabhai wrote:

> Jun 23 14:06:55 nalin-atomic-1.os1.phx2.redhat.com flanneld[697]: 
> E0623 14:06:36.579070 00697 main.go:151] Failed to create 
> SubnetManager: 501: All the given peers are not reachable (Tried to 
> connect to each peer twice and failed) [0]
> Jun 23 14:06:55 nalin-atomic-1.os1.phx2.redhat.com flanneld[697]: 
> E0623 14:06:37.581440 00697 main.go:151] Failed to create 
> SubnetManager: 501: All the given peers are not reachable (Tried to 
> connect to each peer twice and failed) [0]

This error usually means that etcd is not listening on 0.0.0.0 and is
probably only listening on 127.0.0.1.   Or it is only listening on 2379
and you want 4001 (or vice versa)